cmd/kafka-trigger-controller/kafka-controller.go

```diff
@@ -22,12 +22,12 @@ import (
 	"os/signal"
 	"syscall"
 
-	"github.com/kubeless/kafka-trigger/pkg/controller"
-	kafkautils "github.com/kubeless/kafka-trigger/pkg/utils"
-	"github.com/kubeless/kafka-trigger/pkg/version"
 	kubelessutils "github.com/kubeless/kubeless/pkg/utils"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/kubeless/kafka-trigger/pkg/controller"
```

Comment:
I believe these changes in the imports order causes a `gofmt` issue, you'd need to set it back:

```
These files are not properly gofmt'd:
 - cmd/kafka-trigger-controller/kafka-controller.go
 - pkg/controller/kafka_trigger_controller.go
 ```